pkg/util/io/writer.go

```diff
@@ -42,6 +42,11 @@ func (writer *StdWriter) WriteFile(filename string, data []byte, perm os.FileMod
 	return ioutil.WriteFile(filename, data, perm)
 }
 
+const (
+	nsenterPath             = "nsenter"
```

Comment:
As these constants are only used in one method, there is no need to make them as constant vars, just local variables are OK.